tests/0126-oauthbearer_oidc.c

```diff
@@ -33,64 +33,140 @@
 
 
 /**
- * @brief After config OIDC fields, make sure the producer gets created
- *        successfully.
+ * @brief After config OIDC, make sure the producer and consumer
+ *        can work successfully.
  *
  */
-static void do_test_create_producer() {
+static void do_test_produce_consumer_with_OIDC(rd_kafka_conf_t *conf) {
+        if (test_needs_auth()) {
```

Comment:
Doesn't this test actually require auth to be configured?

See comment in main function below.